overlord/snapstate/handlers.go

```diff
@@ -2088,6 +2105,8 @@ func (m *SnapManager) doLinkSnap(t *state.Task, _ *tomb.Tomb) (err error) {
 		}
 	}
 
+	abortMonitoring(st, snapsup.InstanceName())
```

Comment:
this maybe should have a comment to cross reference where the monitoring is setup and what is about, this file is very big